test-app/src/components/debugger.tsx

```diff
@@ -328,7 +329,9 @@ export const Debugger = () => {
       {txId && (
         <Text textStyle="body.large" display="block" my={'base'}>
           <Text color="green" fontSize={1}>
-            Successfully broadcasted &quot;{txType}&quot;
+            <span data-testId={WalletPageSelectors.StatusMessage}>
```

Comment:
This should be moved to the Text component.

Reply:
The text components make the test fail intermittently, the code above makes the test less flaky.

Reply:
@timstackblock Just spotted a capital `I` here.